app/src/main/java/org/kiwix/kiwixmobile/downloader/DownloadService.java

```diff
@@ -132,6 +134,8 @@ public int onStartCommand(Intent intent, int flags, int startId) {
 
     KIWIX_ROOT = checkWritable(KIWIX_ROOT);
```

Comment:
Unrelated to this change, however this code reads oddly - why would checkWritable be used to update KIWIX_ROOT? and KIWIX_ROOT being in all CAPS would lead me to think it's a constant, yet it's being updated. Perhaps something to address in a cleanup CL?